### PR TITLE
Move containerfiles upstream

### PR DESCRIPTION
**Pull request by baldimir**
Adds BAMOE containerfiles and helm chart.

### CODE REVIEW

File: container-images/bamoe-canvas-dev-deployment-dmn-form-webapp/Containerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ARG BAMOE_VERSION_INTERNAL_ARG
+ARG BAMOE_DOCKER_REGISTRY_ARG
+
+FROM --platform=linux/amd64 $BAMOE_DOCKER_REGISTRY_ARG/canvas-dev-deployment-dmn-form-webapp:$BAMOE_VERSION_INTERNAL_ARG
+
+# Custom branding.
+# These are put in place using the script BuildBamoeImage.java
```
Comment (porcelli) [inline container-images/bamoe-canvas-dev-deployment-dmn-form-webapp/Containerfile:7]: what about this?

File: container-images/bamoe-canvas/Containerfile [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+]'
+
+# Removing PMML assets, as they're not part of BAMOE.
+# Disabled until https://github.com/apache/incubator-kie-issues/issues/327 is resolved.
+#RUN rm /var/www/html/samples/Sample.pmml && \
+#    rm /var/www/html/pmml-envelope.html && \
+#    rm /var/www/html/pmml-envelope.js
+
+# Custom branding.
+# These are put in place using the script BuildBamoeImage.java
```
Comment (porcelli) [inline container-images/bamoe-canvas/Containerfile:71]: another reference for BuildBamoeImage.java

**Review by porcelli — CHANGES_REQUESTED** [review]

File: container-images/bamoe-canvas-dev-deployment-dmn-form-webapp/Containerfile (hunk 1 above)
Comment (baldimir) [reply container-images/bamoe-canvas-dev-deployment-dmn-form-webapp/Containerfile]: Fixed.

File: container-images/bamoe-canvas/Containerfile (hunk 2 above)
Comment (baldimir) [reply container-images/bamoe-canvas/Containerfile]: Fixed.

**Review by porcelli — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]
LGTM.

**Pull request merged**